webview/src/plots/components/comparisonTable/styles.module.scss

```diff
@@ -137,6 +122,89 @@ $gap: 4px;
   z-index: 3;
 }
 
+.rowHeader {
+  display: flex;
+  position: sticky;
+  left: 0;
+  width: min-content;
+}
+
+.rowHeaderPath {
+  display: flex;
+  align-items: center;
+
+  .copyButton {
+    display: inline;
+    opacity: 0;
+    margin: 0;
+  }
+
+  &:hover .copyButton {
+    opacity: 1;
+  }
+}
+
+.rowToggler {
+  border: none;
+  background: none;
+  color: $fg-color;
+  text-align: left;
+  display: flex;
+  user-select: text;
+
+  svg {
+    fill: $fg-color;
+    min-width: fit-content;
+  }
+}
+
+.pathText {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  direction: rtl;
+  white-space: nowrap;
+}
+
+.boundingBoxClasses {
+  display: flex;
+  align-items: center;
+  margin-left: 5px;
+}
+
+.boundingBoxClassesTitle {
+  font-size: 0.6rem;
+  margin-right: 10px;
+  color: $fg-color;
+  font-weight: 600;
+}
+
+.hiddenInput {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  border: 0;
+}
```

Comment:
What is the reason for hiding the input this way? (instead of changing its visibility, opacity, or display...)

Reply:
That's fine. I think just making it `position: absolute` and `opacity: 0` is equally accessible, but as long as it isn't shown at all, there's no need to change it.

Reply:
> I think just making it position: absolute and opacity: 0 is equally accessible

While unlikely to happen, it would still be possible to click on the inputs in that case:

https://github.com/iterative/vscode-dvc/assets/43496356/9a9de37f-132e-48d9-830d-f7055be3b3ce

I think I can just set it to `display: none` for simplicity's sake.